packages/sit-onyx/src/components/OnyxVisuallyHidden/OnyxVisuallyHidden.vue

```diff
@@ -0,0 +1,27 @@
+<script lang="ts" setup>
+import type { OnyxVisuallyHiddenProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxVisuallyHiddenProps>(), { is: "span" });
+</script>
+
+<template>
+  <component :is="props.is" class="onyx-visually-hidden"><slot></slot></component>
+</template>
+
+<style lang="scss">
+@use "../../styles/mixins/layers.scss";
+
+.onyx-visually-hidden {
+  @include layers.component() {
+    border: 0;
+    clip: rect(0, 0, 0, 0);
+    height: 0;
+    margin: 0;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
```

Comment:
```suggestion
    width: var(--onyx-1px-in-rem);
```

Reply:
The component should have the smallest possible width, so that it is still recognized as a "visible" element, but it shouldn't scale with the root element font-size.